src/cdk/tree/tree.ts

```diff
@@ -293,6 +315,15 @@ export class CdkTree<T, K = T> implements AfterContentChecked, CollectionViewer,
       CdkTreeNode.mostRecentTreeNode.data = nodeData;
     }
   }
+
+  /** Level accessor, used for compatibility between the old Tree and new Tree */
+  get getLevel() {
+    return this.treeControl?.getLevel ?? this.levelAccessor;
+  }
+
+  get getChildren() {
+    return this.treeControl?.getChildren ?? this.childrenAccessor;
+  }
```

Comment:
Could these be private methods? In general we avoid getters unless necessary for Angular inputs or backwards compatibility. It also looks like they're not used outside of this file

Reply:
yep, done